Example/TouchID/ViewController.swift

```diff
@@ -18,6 +18,12 @@ public final class ViewController: UIViewController {
     // MARK: - Private properties -
 
     private let identifier = "TouchIDSampleApp"
+
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+        Locker.enableDeviceListSync = true
+
+    }
```

Comment:
As this is the example project, consider moving this into the AppDelegate `didFinishLaunchingWithOptions`